tools/waffle-example/README.md

```diff
@@ -0,0 +1,72 @@
+# Waffle example
+
+Simple scripts for testing matchers provided by Waffle tool.
+
+## Configuration
+
+Create `.env` file based on `.env.example`
+
+```
+OPERATOR_PRIVATE_KEY=
+RELAY_ENDPOINT=
+```
+
+ - `OPERATOR_PRIVATE_KEY` is your account ECDSA hex-encoded private key.
+ - `RELAY_ENDPOINT` is a path to your JSON RPC Api. `https://testnet.hashio.io/api` for testnet.
```

Comment:
Add a remainder to start local node when using the endpoint `http://localhost:7546`.